target_ws_pkgs.repos

```diff
@@ -147,12 +149,12 @@ repositories:
   extra/ros-controls/control_msgs:
     type: git
     url: https://github.com/ros-controls/control_msgs.git
-    # technically on 'rolling', but this seems the version released into Iron ..
-    version: 5.0.0
+    # technically on 'rolling', but this is the version released into Iron ..
+    version: 5.2.0
   extra/ros2/geometry2_tf2_msgs:
     type: git
     url: https://github.com/ros2/geometry2.git
-    version: 0.31.5
+    version: 0.31.7
```

Comment:
we only use `tf2_msgs` from this, and both `0.31.6` and `0.31.7` don't contain any changes to that package ([here](https://github.com/ros2/geometry2/commit/719c3a212480f8f418ba682ba53c623687bb366d#diff-ca78c11f23eece3ef5739c290f99b57bff30c3d7c44e630056580cc52de3701e) and [here](https://github.com/ros2/geometry2/commit/c2b74336100e5a24720dccaac102f181511bd7dd#diff-ca78c11f23eece3ef5739c290f99b57bff30c3d7c44e630056580cc52de3701e)).